We disable security, because chroot() does not work with our environment.

With chdir() this can be used for bootstrapping on a local network.

diff -u -r ../busybox-1.22.1/networking/tftp.c ./networking/tftp.c
--- ../busybox-1.22.1/networking/tftp.c	2014-01-09 19:15:44.000000000 +0100
+++ ./networking/tftp.c	2014-07-01 08:42:48.000000000 +0200
@@ -790,7 +790,7 @@
 		G.pw = xgetpwnam(user_opt);
 	}
 	if (argv[0]) {
-		xchroot(argv[0]);
+		xchdir(argv[0]);
 	}
 
 	result = recv_from_to(STDIN_FILENO, G.block_buf, sizeof(G.block_buf),
